src/main/java/com/foundation/search/view/ParametersPanel.java

```diff
@@ -317,7 +360,128 @@ public JButton getCleanButton(){
         return cleanButton;
     }
 
-     /**
+    /**
+     * This method initialize all input labels, combo boxes and check boxes
+     * displayed in this panel
+     *
+     */
+    public void initializeInputs(){
+        //initializing the inputs, comboboxes and checkboxes
+        nameInput = new JTextField();
+        String[] extensions = {" ","exe", "png","mp4","pdf","mp4","jpg","txt","xls",
+                               "xcls","sh","java","php","bat","avi"};
+        extensionInput = new JComboBox(extensions);
+        sizeInput = new JTextField();
+        String[] comparisonSelection = {" ", "<", ">","="};
```

Comment:
Please remove unnecessary space between quotation marks.
Same comment for units.

Reply:
done, thanks